### PR TITLE
Brain on blue

### PR DESCRIPTION
**Pull request by Rus604**
proposal is included in the branch

### CODE REVIEW

File: Project_Proposal.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### List of game features:
+* side scrolling adventure
+* interesting obstacles to navigate with left right and jump
+* brain stem is an attack, "down key"
+* at least two levels , and second level is "moving"faster
+
+### Realistic goals/expectations for 4 weeks of work on the project:
+without a "cool background, it should be do-able, the level may be 
+the only possible constraint.
```
Comment (SamBumgardner) [inline Project_Proposal.md:27]: Making levels doesn't have to be too hard, I'll make sure we talk about it soon.

File: Project_Proposal.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+### Lose condition:
+Obstacles cause enough damage that BOB dies
+
+### How the player interacts with the game:
+the interface will be via the keyboard using the "up", "Down",
+"left", and right arrow keys.
+
+### List of game features:
+* side scrolling adventure
+* interesting obstacles to navigate with left right and jump
```
Comment (SamBumgardner) [inline Project_Proposal.md:21]: Might be good to elaborate on obstacle types. Are you just wanting hazards that damage/kill the player or do you want moving platforms, conveyor belts, or something else?

Having a list of specific obstacle types you want will help you figure out what is actually feasible for you to implement.

File: Project_Proposal.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* brain stem is an attack, "down key"
```
Comment (SamBumgardner) [inline Project_Proposal.md:22]: What can BOB attack?

File: Project_Proposal.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+### Lose condition:
+Obstacles cause enough damage that BOB dies
+
+### How the player interacts with the game:
+the interface will be via the keyboard using the "up", "Down",
+"left", and right arrow keys.
+
+### List of game features:
+* side scrolling adventure
```
Comment (SamBumgardner) [inline Project_Proposal.md:20]: By this, do you mean the game is always automatically scrolling?

If you do want to do this, there will be a few things extra things to worry about during programming. Specifically, we need to figure out how BOB will interact with the edge of the screen.
 * Does it push him along?
 * If he's trapped between the edge of the screen and a wall, is he crushed?

It'll also involve actually programming the invisible wall and making sure the game camera moves at the same speed (probably following some invisible FlxObject at screen center).

**Review by SamBumgardner — COMMENTED** [review]
I definitely think this game is doable. Good work on making a proposal that has limited (and thus realistic) scope.

That said, I think it'd be good consider and answer the questions I posed in the body of the PR. Figuring out the answers to those questions will hopefully help eliminate any surprise questions you might encounter later in code.

File: Project_Proposal.md (hunk 1 above)
Comment (Rus604) [reply Project_Proposal.md:27]: I feel i might need to go back over level creation when i get to thet part of development

File: Project_Proposal.md (hunk 4 above)
Comment (Rus604) [reply Project_Proposal.md:20]: we spoke about having the obstacles have their own velocity across the screen, I will be using that in the development

**Comment by Rus604** [comment]
I have created a new Repo , Brain-On-Blue,  with the changes entered into the proposal.  I will be creating a new push request from the new repo

**Pull request closed without merging**